files/galaxy/dynamic_job_rules/production/total_perspective_vortex/tools.yml

```
@@ -270,16 +270,18 @@ tools:
     cores: 6
   toolshed.g2.bx.psu.edu/repos/galaxyp/maxquant/maxquant/.*:
     cores: 16
-    context:
-      partition: max_quant_w8
+    rules:
+    - if: |
+        user_limit = 2
+        total_limit = 4
+        helpers.concurrent_job_count_for_tool(app, tool, user) >= user_limit or helpers.concurrent_job_count_for_tool(app, tool) >= total_limit
```

This conditional seems to be common enough that perhaps we should move it into tpv core with something like:
```suggestion
        helpers.do_concurrent_jobs_exceed_limits(app, tool, user, user_limit=2, global_limit=4)
```
That function could handle return true or false, depending on whether only user_limit, global_limit or both are defined.

---

That's a good idea and saves it from doing the job query twice

---

I'd like to merge this as is since we are still on v1.x